build.sbt

```diff
@@ -83,7 +83,7 @@ val googleCloudDatastoreVersion = "0.108.6"
 val googleCloudMonitoringVersion = "3.32.0"
 val googleCloudPubSubVersion = "1.107.13"
 val googleCloudSpannerVersion = "6.55.0"
-val googleCloudStorageVersion = "2.30.1"
+val googleCloudStorageVersion = "2.26.0"
```

Comment:
in sync with [Beam](https://storage.googleapis.com/cloud-opensource-java-dashboard/com.google.cloud/libraries-bom/26.22.0/index.html)

Reply:
Your link points to GCP libraries-bom 26.22.0 but beam 2.53 uses 26.28.0 [here](https://github.com/apache/beam/blob/260b554a007725c9277c98f5ec8ef704570a238a/buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy#L757C87-L757C94).

Reply:
Right, reverting this